student.rb

```diff
@@ -1,7 +1,5 @@
 require './person'
 class Student < Person
-  attr_accessor :classroom
```

Comment:
- Hey, in here also, you should be able to read the value of the classroom in your attr_reader, then instead of checking in this condition `classroom.students.push(self) unless classroom.students.include?(self)`, you'd rather just push 
`classroom.students.push(self)` as you've established this in reading it in your setter/getter method.